conans/client/cmd/export_pkg.py

```diff
@@ -58,8 +58,13 @@ def export_pkg(cache, graph_manager, hook_manager, recorder, output,
     readed_manifest = FileTreeManifest.load(dest_package_folder)
     pref = PackageReference(pref.ref, pref.id, readed_manifest.summary_hash)
     output.info("Package revision %s" % pref.revision)
+
     with layout.update_metadata() as metadata:
         metadata.packages[package_id].revision = pref.revision
         metadata.packages[package_id].recipe_revision = metadata.recipe.revision
 
+    if graph_info.graph_lock:
+        # after the package has been created we need to update the node PREV
+        nodes[0].prev = pref.revision
+        graph_info.graph_lock.update_check_graph(deps_graph, output)
```

Comment:
I think this is not covered by the tests. Only RREV but not PREV

Reply:
Added test. The ``export-pkg`` creates a new binary, so it is able to modify both the RREV if the exported recipe changes, and for sure the PREV is being updated too because of the new binary.